lib/Interpreter/Functions/Template/Mustache.php

```diff
@@ -12,12 +12,19 @@
  * Returns the resulting string.
```

Comment:
Its not necessary to update the `docs/` markdown files, can you instead add the changes to this doc comment, I use these to generate the markdown files on each release.